docs/sources/github/example-api-responses/sanitized/commit.json

```diff
@@ -7,11 +7,11 @@
   "commit":{
     "url":"https://api.github.com/repos/org/Hello-World/git/commits/6dcb09b5b57875f334f61aebed695e2e4193db5e",
     "author":{
-      "email":"{\"scope\":\"github\",\"hash\":\"T-sBXXi6k0bWw-zNtcdQenxAuqN5C51_ehZ5lJDWeqw\"}",
+      "email":"{\"scope\":\"email\",\"domain\":\"github.com\",\"hash\":\"39uRnKxs0PYIueRoGGc7fyyp-hSTtamEhvB6YywFNVM\"}",
```

Comment:
This is what it was wrong, the double pseudonymization generates an invalid identity, considered it as *github* when it is actually an email from someone